po/tr.po

```diff
@@ -0,0 +1,159 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```

Comment:
Take a look at the [English translation](https://github.com/xou816/spot/blob/master/po/en.po) to see how to fill this in.
https://github.com/xou816/spot/blob/552ad4503180b5dcff5de8c5bb877c4e4afe5b68/po/en.po#L1-L4